modules/oe_theme_content_event/src/Plugin/ExtraField/Display/DescriptionExtraField.php

```diff
@@ -92,13 +92,13 @@ public function viewElements(ContentEntityInterface $entity) {
     $build = [
       '#type' => 'pattern',
       '#id' => 'text_featured_media',
-      '#fields' => [
-        'title' => $this->getRenderableTitle($entity),
-        'text' => $this->getRenderableText($entity),
-      ],
     ];
-
+    // If we have no renderable text and image wee don't need a title.
     $this->addFeaturedMediaThumbnail($build, $entity);
+    $text = $this->getRenderableText($entity);
+    $title = !empty($text[0]['#text']) || !empty($build['#fields']['image']) ? $this->getRenderableTitle($entity) : '';
+    $build['#fields']['title'] = $title;
+    $build['#fields']['text'] = $text;
```

Comment:
This is not good. When there is no media or description we're rendering an empty pattern with cache tags.
<img width="303" alt="text field" src="https://user-images.githubusercontent.com/22908988/198243134-b54f7417-fb21-4c7a-9d79-d52750652334.png">

If we don't have a media or text we should simply return an empty array instead of rendering a pattern template that's empty.
<img width="908" alt="empty markup" src="https://user-images.githubusercontent.com/22908988/198242793-2d5f638b-c7b7-4a40-81ab-512ce1801464.png">